### ISSUE TITLE
Different installation issue

### ISSUE THREAD

**Issue by angelajmcd** [holmrenser/IOGA#8]
Hi Rens,

I'm trying to use IOGA on my mac. When I run setup_IOGA.py, it looks like there is an issue with installing samtools 0.1.19 and I'm not sure how to fix it.

Any ideas? 
Angela

The output from running the script is below. (It doesn't make the .json file.)

picard.jar
100% [......................................................] 7353972 / 7353972
chmod +x picard.jar
trying /Applications/IOGA/exe/picard-tools-1.124/picard.jar
succes
samtools
100% [........................................................] 514507 / 514507x samtools-0.1.19/
x samtools-0.1.19/.gitignore
x samtools-0.1.19/AUTHORS
x samtools-0.1.19/COPYING
x samtools-0.1.19/ChangeLog.old
x samtools-0.1.19/INSTALL
x samtools-0.1.19/Makefile
x samtools-0.1.19/Makefile.mingw
x samtools-0.1.19/NEWS
x samtools-0.1.19/bam.c
x samtools-0.1.19/bam.h
x samtools-0.1.19/bam2bcf.c
x samtools-0.1.19/bam2bcf.h
x samtools-0.1.19/bam2bcf_indel.c
x samtools-0.1.19/bam2depth.c
x samtools-0.1.19/bam_aux.c
x samtools-0.1.19/bam_cat.c
x samtools-0.1.19/bam_color.c
x samtools-0.1.19/bam_endian.h
x samtools-0.1.19/bam_import.c
x samtools-0.1.19/bam_index.c
x samtools-0.1.19/bam_lpileup.c
x samtools-0.1.19/bam_mate.c
x samtools-0.1.19/bam_md.c
x samtools-0.1.19/bam_pileup.c
x samtools-0.1.19/bam_plcmd.c
x samtools-0.1.19/bam_reheader.c
x samtools-0.1.19/bam_rmdup.c
x samtools-0.1.19/bam_rmdupse.c
x samtools-0.1.19/bam_sort.c
x samtools-0.1.19/bam_stat.c
x samtools-0.1.19/bam_tview.c
x samtools-0.1.19/bam_tview.h
x samtools-0.1.19/bam_tview_curses.c
x samtools-0.1.19/bam_tview_html.c
x samtools-0.1.19/bamshuf.c
x samtools-0.1.19/bamtk.c
x samtools-0.1.19/bcftools/
x samtools-0.1.19/bcftools/Makefile
x samtools-0.1.19/bcftools/README
x samtools-0.1.19/bcftools/bcf.c
x samtools-0.1.19/bcftools/bcf.h
x samtools-0.1.19/bcftools/bcf.tex
x samtools-0.1.19/bcftools/bcf2qcall.c
x samtools-0.1.19/bcftools/bcfutils.c
x samtools-0.1.19/bcftools/call1.c
x samtools-0.1.19/bcftools/em.c
x samtools-0.1.19/bcftools/fet.c
x samtools-0.1.19/bcftools/index.c
x samtools-0.1.19/bcftools/kfunc.c
x samtools-0.1.19/bcftools/kmin.c
x samtools-0.1.19/bcftools/kmin.h
x samtools-0.1.19/bcftools/main.c
x samtools-0.1.19/bcftools/mut.c
x samtools-0.1.19/bcftools/prob1.c
x samtools-0.1.19/bcftools/prob1.h
x samtools-0.1.19/bcftools/vcf.c
x samtools-0.1.19/bcftools/vcfutils.pl
x samtools-0.1.19/bcftools/bcf.h~
x samtools-0.1.19/bedcov.c
x samtools-0.1.19/bedidx.c
x samtools-0.1.19/bgzf.c
x samtools-0.1.19/bgzf.h
x samtools-0.1.19/bgzip.c
x samtools-0.1.19/cut_target.c
x samtools-0.1.19/errmod.c
x samtools-0.1.19/errmod.h
x samtools-0.1.19/examples/
x samtools-0.1.19/examples/00README.txt
x samtools-0.1.19/examples/Makefile
x samtools-0.1.19/examples/bam2bed.c
x samtools-0.1.19/examples/calDepth.c
x samtools-0.1.19/examples/chk_indel.c
x samtools-0.1.19/examples/ex1.fa
x samtools-0.1.19/examples/ex1.sam.gz
x samtools-0.1.19/examples/toy.fa
x samtools-0.1.19/examples/toy.sam
x samtools-0.1.19/faidx.c
x samtools-0.1.19/faidx.h
x samtools-0.1.19/kaln.c
x samtools-0.1.19/kaln.h
x samtools-0.1.19/khash.h
x samtools-0.1.19/klist.h
x samtools-0.1.19/knetfile.c
x samtools-0.1.19/knetfile.h
x samtools-0.1.19/kprobaln.c
x samtools-0.1.19/kprobaln.h
x samtools-0.1.19/kseq.h
x samtools-0.1.19/ksort.h
x samtools-0.1.19/kstring.c
x samtools-0.1.19/kstring.h
x samtools-0.1.19/misc/
x samtools-0.1.19/misc/HmmGlocal.java
x samtools-0.1.19/misc/Makefile
x samtools-0.1.19/misc/ace2sam.c
x samtools-0.1.19/misc/bamcheck.c
x samtools-0.1.19/misc/blast2sam.pl
x samtools-0.1.19/misc/bowtie2sam.pl
x samtools-0.1.19/misc/export2sam.pl
x samtools-0.1.19/misc/interpolate_sam.pl
x samtools-0.1.19/misc/maq2sam.c
x samtools-0.1.19/misc/md5.c
x samtools-0.1.19/misc/md5.h
x samtools-0.1.19/misc/md5fa.c
x samtools-0.1.19/misc/novo2sam.pl
x samtools-0.1.19/misc/plot-bamcheck
x samtools-0.1.19/misc/psl2sam.pl
x samtools-0.1.19/misc/r2plot.lua
x samtools-0.1.19/misc/sam2vcf.pl
x samtools-0.1.19/misc/samtools.pl
x samtools-0.1.19/misc/soap2sam.pl
x samtools-0.1.19/misc/varfilter.py
x samtools-0.1.19/misc/vcfutils.lua
x samtools-0.1.19/misc/wgsim.c
x samtools-0.1.19/misc/wgsim_eval.pl
x samtools-0.1.19/misc/zoom2sam.pl
x samtools-0.1.19/padding.c
x samtools-0.1.19/phase.c
x samtools-0.1.19/razf.c
x samtools-0.1.19/razf.h
x samtools-0.1.19/razip.c
x samtools-0.1.19/sam.c
x samtools-0.1.19/sam.h
x samtools-0.1.19/sam_header.c
x samtools-0.1.19/sam_header.h
x samtools-0.1.19/sam_view.c
x samtools-0.1.19/sample.c
x samtools-0.1.19/sample.h
x samtools-0.1.19/samtools.1
x samtools-0.1.19/win32/
x samtools-0.1.19/win32/libcurses.a
x samtools-0.1.19/win32/libz.a
x samtools-0.1.19/win32/xcurses.h
x samtools-0.1.19/win32/zconf.h
x samtools-0.1.19/win32/zlib.h
x samtools-0.1.19/bam.h~

**Comment by holmrenser** [comment]
I'm looking into this! Maybe it has something to do with the samtools/mac combination, but I'm not quite sure yet...

**Comment by holmrenser** [comment]
Update: I fixed the error you mentioned, it was due to the tar command in the setup script. Unfortunately I found out that SOAPdenovo will not run on a macbook, so it seems it is currently not possible to use IOGA on a mac...

I will look into a way to fix this next week, either by compiling SOAPdenovo for a mac (if that is possible), or by switching it out to another assembler (for instance minia).

In the meantime: if you manage to compile SOAPdenovo on your mac yourself and point to that binary in the IOGA_config.json file that is created by the setup_IOGA.py script, that might also be a fix.
